packages/components/buttons/secondary-button/src/secondary-button.visualroute.jsx

```diff
@@ -93,6 +93,14 @@ export const component = () => (
       <SecondaryButton label="A label text" onClick={() => {}} size="medium" />
     </Spec>
 
+    <Spec label='size - when "20"'>
+      <SecondaryButton label="A label text" onClick={() => {}} size="big" />
```

Comment:
```suggestion
      <SecondaryButton label="A label text" onClick={() => {}} size="20" />
```

Reply:
I left this intentionally, so that we can visually verify that the mapping (big->20) is leading to the same result as before. I added the new sizes below.